config/locales/en.yml

```diff
@@ -31,3 +31,9 @@
 
 en:
   hello: "Hello world"
```

Comment:
Lets remove the boilerplate code 🙈

Reply:
Fixed in 366f0e361c9209f473a876ffbab0e837add6eb33